app/policies/roles/institution/health_organization.json.erb

```diff
@@ -10,20 +10,14 @@
             "institution:read",
             "institution:update",
             "institution:delete",
-            "institution:registerDevice",
-            "institution:createSite",
-            "institution:createRole"
+            "institution:createRole",
+            "institution:readUsers"
           ],
           "delegable": true
         },
         {
-          "resource": [
-            "site?institution=<%= institution_id %>",
-            "device?institution=<%= institution_id %>",
-            "testResult?institution=<%= institution_id %>",
-            "encounter?institution=<%= institution_id %>"
-          ],
-          "action": "*",
+          "resource": "testResult?institution=<%= institution_id %>",
+          "action": "testResult:medicalDashboard",
```

Comment:
Also, note that NNDD currently does not support users having access to tests but not encounters. If we do have roles with this difference, then we'll need to push the permissions info to NNDD somehow.